llvm/test/MC/ARM/pcrel-thumb-ldrd-relocs.s

```diff
@@ -0,0 +1,6 @@
+@ RUN: not llvm-mc -filetype=obj -o /dev/null %s 2>&1 -triple=thumbv7   | FileCheck %s
+@ RUN: not llvm-mc -filetype=obj -o /dev/null %s 2>&1 -triple=thumbebv7 | FileCheck %s
+
+   ldrd r0, r1, foo
```

Comment:
Is this related to positive tests in another test file? If true, we can merge the test files by utilizing `.ifdef ERR` and `--defsym ERR=1`. Grep will a few examples.

Reply:
Thanks for the tip. I've merged `pcrel-thumb-ldrd-relocs.s` and `pcrel-arm-ldrd-relocs.s` into `pcrel-ldrd-diff-section.s`